packages/main/types/content.ts

```diff
@@ -54,6 +54,55 @@ export interface TextPart {
 export interface InlineDataPart {
   text?: never;
   inlineData: GenerativeContentBlob;
+  functionCall?: never;
+  functionResponse?: never;
+}
+
+/**
+ * Content part interface if the part represents FunctionResponse.
```

Comment:
Reminder to run `yarn docs` to generate reference md files with these new interfaces.